lib/new.js

```diff
@@ -160,19 +161,54 @@ const promptForName = (args) => {
     return value;
   };
 
-  // No need to check for err since we're using a validator
-  const handler = (err, value) => {
-    settings.spa = value;
-    settings.name = 'skyux-spa-' + value;
-    settings.path = path.join('.', settings.name);
-    settings.pathTmp = path.join(settings.path, 'tmp');
-    promptForUrl();
-  };
+  return promptly.prompt(prompt, { validator: validator })
+    .then((value) => {
+      settings.spa = value;
+      settings.name = 'skyux-spa-' + value;
+      settings.path = path.join('.', settings.name);
+      settings.pathTmp = path.join(settings.path, 'tmp');
+      return Promise.resolve(value);
+    });
+};
+
+/**
+ * Returns an object representing the template config, derived
+ * from constructor arguments.
+ * @name getTemplateFromArgs
+ */
+const getTemplateFromArgs = (args) => {
```

Comment:
I'll rework that method, but the `args` argument is `undefined` when run through the tests.